topsort-endpoints.yml

```diff
@@ -115,8 +115,18 @@ components:
           minItems: 1
           items:
             $ref: '#/components/schemas/Product'
+        brands:
+          type: array
+          description: An array of objects, each describing a brand that should participate in the auction.
+          minItems: 1
```

Comment:
This should not have a minimum of 1. As you could have banner ads even though you may not have the concept of brands.